src/librustc_trans/trans/builder.rs

```diff
@@ -254,6 +263,15 @@ impl<'a, 'tcx> Builder<'a, 'tcx> {
         }
     }
 
+    pub fn fsub_fast(&self, lhs: ValueRef, rhs: ValueRef) -> ValueRef {
+        self.count_insn("sub");
```

Comment:
That seems logical, this just repeats what `fsub` did above, both should be fixed.